### PR TITLE
Add homepage-bootstrap

### PR DESCRIPTION
**Pull request by pbkabali**
### What this PR does

- Adds the homepage that has been designed with bootstrap

### Screenshot

image

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  top: 0;
+  right: 0;
+}
+
+.fa-facebook-official,
+.fa-twitter,
+.fa-instagram,
+.fa-tumblr,
+.fa-linkedin {
+  font-size: 1.5rem !important;
```
Comment (elmejdki) [inline styles.css]: - please remove all the important keywords

**Review by elmejdki — CHANGES_REQUESTED** [review]
# status: require changes ♻️ 
## Great job Guys 👏 I couldn't tell the difference between the original and your clone, but there are some minor changes 🔨 🔧 🧰  that you need to fix in order to make it perfect 👍 💯 

- please center the newsletter form is not centered for the mobile and tablet views.

- add some margin or padding on the bottom of the page so the footer can be seen easily like in the original page.

- center the subscribe text as centered in the original page:

|*Clone*|*Original*|
|:---:|:---:|
| image | image |

- [x] Fix the header on mobile view it's not showing correctly:

|*Clone*|*Original*|
|:---:|:---:|
| image | image |

### Please make all the required changes before you submit another code review.
## Happy Coding 🎩

File: styles.css (hunk 1 above)
Comment (Miguelus373) [reply styles.css]: Thank you @elmejdki for your feedback!
We worked on the changes and we'll request another review.

**Review by kubilaycaglayan — APPROVED** [review]
Hello Team :wave: 
### Status: Approved :green_circle: 

Amazing work! Great job on doing all the required changes.
Thanks for all of your efforts... :star: :chart_with_upwards_trend: 

With all my respect.

**Pull request merged**